README.md

```diff
@@ -74,6 +74,46 @@ metadata:
   name: dpdk-checkup-traffic-gen-sa
 ```
 
+Apply the following `SecurityContextConstraints` object:
```

Comment:
I think it's worth adding a section for additional configuration required for running it on Openshift, or, if not a new section, at least an explanation that it's required for Openshift clusters.